cost/azure/idle_compute_instances/azure_idle_compute_instances.pt

```diff
@@ -488,20 +492,32 @@ script "js_instance_cost_mapping", type:"javascript" do
   EOS
 end
 
+script "js_only_instances", type: "javascript" do
+  parameters "arg"
+  result "results"
+  code <<-EOS
+    results = arg.instance_list
+    if (results && results.length) {
+      results[0].message = arg.message
+    }
+  EOS
+end
+
 ###############################################################################
 # Policy
 ###############################################################################
 
 policy "azure_resource_policy" do
-  validate $ds_instance_cost_mapping do
+  validate $ds_only_instances do
     summary_template "{{ rs_project_name }} (Account ID: {{ rs_project_id }}): Azure idle compute instances found"
+    hash_exclude "averageCPU", "averages", "savings"
```

Comment:
What is "averages" here referring to? This isn't an entry in the export statement.

Reply:
In tests, adding it to the hash_exclude, the results have been successful, when I remove it, the incidents continue to be generated.

Is a field contained in the object but not exported
<img width="369" alt="image" src="https://github.com/flexera-public/policy_templates/assets/22282464/5812835d-5861-4b7e-9135-042891c0fde2">